girder/utility/assetstore_utilities.py

```diff
@@ -62,6 +62,16 @@ def getAssetstoreAdapter(assetstore, instance=True):
 
 
 def setAssetstoreAdapter(storeType, cls):
+
+    """
+    This updates the internal assetstore adapter table with either a new entry,
+    or a modification to an existing entry. Subsequent calls to
+    getAssetstoreAdapter() will return the modified class (or instance thereof),
+    allowing for dynamic updating of assetstore behavior at runtime.
+
+    :param storeType: The assetstore type to create/modify.
+    :type storeType: enum | any
```

Comment:
Could we also document the `cls` arg, in particular mention that it should be a subclass of `AbstractAssetstoreAdapter`.